rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala

```diff
@@ -573,7 +646,15 @@ class ProcPrinterSpec extends FlatSpec with Matchers {
     val result =
       PrettyPrinter().buildString(
         ProcNormalizeMatcher.normalizeMatch[Coeval](basicInput, inputs).value.par)
-    result shouldBe "match (47 == 47) { true => new x0 in { x0!(47) } ; false => new x0 in { x0!(47) } }"
+    result shouldBe
+      """match (47 == 47) {
+        |\u0009true => new x0 in {
```

Comment:
You should be able to just use tab instead of unicode symbol.

Reply:
1. In scala, multi-line String doesn't support escape characters. 
2. It is difficult to input tab directly in IDEA, though I could do that way. But I think it is worse to maintain than this way.

Reply:
Just to be clear - please do use the 2 spaces instead of tabs all around.